### PR TITLE
fix: incorrect fee

### PR DESCRIPTION
**Pull request by vivalaakam**
## Proposed Changes

- incorrect fee

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

**Review by ragozin-nikita — COMMENTED** [review]
Please add test for this new feature

File: src/components/transaction-detail.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ export const TransactionDetail = ({
   }, [transaction, source, isSent]);
 
   const fee = useMemo(
-    () => new Balance(transaction.feeHex || transaction.fee).toWeiString(),
+    () => new Balance(transaction.feeHex || transaction.fee).toBalanceString(8),
```
Comment (ragozin-nikita) [inline src/components/transaction-detail.tsx:88]: Please create a constant instead of using magic numbers

File: ios/haqq/Info.plist [hunk 2]
```diff
@@ -93,6 +93,11 @@
 	<dict>
 		<key>NSExceptionDomains</key>
 		<dict>
+			<key>45.77.65.60</key>
```
Comment (ragozin-nikita) [inline ios/haqq/Info.plist:96]: What is this?

File: src/screens/settings-test.tsx [hunk 3]
```diff
@@ -391,9 +391,9 @@ export const SettingsTestScreen = observer(() => {
     const unsignedTx = await EthNetwork.populateTransaction(
       wallet.address,
       contract,
-      new BN(100000000000000),
+      new Balance(100000000000000, 0),
       data,
-      250000,
+      new Balance(250000, 0),
```
Comment (ragozin-nikita) [inline src/screens/settings-test.tsx:396]: What is 0 precision and why?

File: src/services/balance.ts [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
 
 const zeroBN = new Decimal(0);
 
 export class Balance implements IBalance {
   static readonly Empty = new Balance(zeroBN);
   private bnRaw = zeroBN;
+  private _precission: number;
 
-  constructor(balance: BalanceConstructor) {
+  constructor(balance: BalanceConstructor, precission = 18) {
```
Comment (ragozin-nikita) [inline src/services/balance.ts:15]: Same magic number here

**Review by ragozin-nikita — CHANGES_REQUESTED** [review]

File: src/screens/settings-test.tsx (hunk 3 above)
Comment (vivalaakam) [reply src/screens/settings-test.tsx:396]: not to write new Balance(0.00000000000025)

File: ios/haqq/Info.plist (hunk 2 above)
Comment (vivalaakam) [reply ios/haqq/Info.plist:96]: developer node without https

**Review by ragozin-nikita — APPROVED** [review]

**Pull request merged**